internal/venice-common/src/main/proto/VeniceReadService.proto

```diff
@@ -0,0 +1,27 @@
+syntax = 'proto3';
+package com.linkedin.venice.protocols;
+
+option java_multiple_files = true;
+
+service VeniceReadService {
+  rpc get (VeniceClientRequest) returns (VeniceServerResponse) {}
+  rpc batchGet(VeniceClientRequest) returns (VeniceServerResponse) {}
+}
+
+message VeniceClientRequest {
+  uint32 partition = 1;
+  string keyString = 2;           // used for single get
+  bytes  keyBytes = 3;            // used for batch get
+  sint32 requestTimeoutInNS = 4;
+  string resourceName = 5;
+  string storeName = 6;
+  bool   isStreamingRequest = 7;
+  bool   isRetryRequest = 8;
+}
+
+message VeniceServerResponse {
```

Comment:
It seems this definition doesn't include compression header here.

Reply:
we have updated it to include the compression value.